modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -70,32 +62,32 @@ The bootstrap script is also available at [path]``https://example.com/pub/bootst
 In this section you will copy and modify the template bootstrap script you created from pass:c[xref:FILENAME.adoc#generate.bootstrap.script[]].
 
 A minimal requirement when modifying a bootstrap script for use with {productname} is the inclusion of an activation key.
-Depending on your organizations security requirements it is strongly recommended to include one or more (GPG) keys (for example, your organization key, and package signing keys).
-For this tutorial you will be registering with the activation keys created in the previous section.
+Depending on your organization's security requirements it is strongly recommended to include one or more GPG keys (for example, your organization key, and package signing keys).
```

Comment:
This phrase is not really correct. It is not about "organization's security requirements".

If you try to install a signed package where the system does not "trust" the signing key, it will ask the user if he really want to install that package.

But as we talk here about "non-interactive" processes, the default answer to this question is "no".
So not having the GPG keys trusted on the system prevent installing and updating packages.

By default every system trust the vendor GPG key. As long as you install only packages from vendor you do not need to trust additional keys. But when you start to install packages signed with your own key or RPMs from a different vendor (e.g. SUSE Client Tools packages on RedHat or Ubuntu) you need to trust the GPG keys which were used to sign these packages.

Reply:
Trying to capture that in a short sentence is hard. How's this?

> Most packages are signed with GPG, so you will also need to have trusted GPG keys on your system to install them.

Reply:
Sounds good.